nativelink-config/src/stores.rs

```diff
@@ -313,20 +311,18 @@ pub enum StoreSpec {
     /// **Example JSON Config:**
     /// ```json
     /// "existence_cache": {
-    ///     "backend": {
-    ///       "memory": {
-    ///         "eviction_policy": {
-    ///           // 500mb.
-    ///           "max_bytes": 500000000,
-    ///         }
-    ///       }
-    ///     },
-    ///     "cas_store": {
-    ///       "ref_store": {
-    ///         "name": "CAS_MAIN_STORE"
+    ///   "backend": {
+    ///     "memory": {
+    ///       "eviction_policy": {
+    ///         "max_bytes": "500mb",
     ///       }
     ///     }
+    ///   },
+    ///   // Note this is the existence store policy, not the backend policy
+    ///   "eviction_policy": {
+    ///     "max_seconds": "100",
```

Comment:
```suggestion
    ///     "max_seconds": 100,
```

as per `web/platform/src/content/docs/docs/config/production-config.mdx`